src/LocalFile.js

```diff
@@ -1,85 +1,91 @@
+"use strict";
```

Comment:
Babel already adds this for us when we transpile: [for example.](https://github.com/njosefbeck/gatsby-source-stripe/blob/7195fe1aa7a22483f72727f42d7df13f022b82bf/LocalFile.js#L1).